### PR TITLE
containerregistry v0.0.23 -> v0.0.24

### PR DESCRIPTION
**Pull request by dekkagaijin**
puller and importer checksums generated by:
`curl https://storage.googleapis.com/containerregistry-releases/v0.0.24/(puller, importer).par | sha256sum`

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by dekkagaijin** [comment]
I signed it!

**Comment by dekkagaijin** [comment]
@mattmoor ptal

**Comment by dekkagaijin** [comment]
I signed it!

**Comment by mattmoor** [comment]
Test this please

(for Bazel CI)

**Comment by mattmoor** [comment]
Please incorporate that internal patch I sent you, which should make `container_flatten` considerably faster.

**Comment by mattmoor** [comment]
Test this please.

File: container/flatten.bzl [hunk 1]
```diff
@@ -32,24 +32,30 @@ def _impl(ctx):
   legacy_base_arg = []
   legacy_files = []
   if image.get("legacy"):
-    # TODO(mattmoor): warn about legacy base.
+
```
Comment (mattmoor) [inline container/flatten.bzl]: Remove blank line

Comment (mattmoor) [reply container/flatten.bzl]: Can you remove the `_get_runfile_path` import above too?

**Comment by dekkagaijin** [comment]
Done.

**Comment by mattmoor** [comment]
Test this please

**Pull request merged**